ale_linters/elm/make.vim

```diff
@@ -32,10 +37,10 @@ function! ale_linters#elm#make#GetCommand(buffer) abort
         let l:dir_set_cmd = ''
     else
         let l:root_dir = fnamemodify(l:elm_package, ':p:h')
-        let l:dir_set_cmd = 'cd ' . fnameescape(l:root_dir) . '; '
+        let l:dir_set_cmd = 'cd ' . fnameescape(l:root_dir) . ' && '
     endif
 
-    let l:elm_cmd = 'elm-make --report=json --output='.shellescape(g:ale#util#nul_file)
+    let l:elm_cmd = 'elm-make --report=json --output='.shellescape('/dev/null')
```

Comment:
Will this break on Windows? I wonder if we should just remove the `shellescape` call where. We know the strings will be either `/dev/null` for Unix system or the special filename `nul` for Windows, which is the Windows equivalent.